.github/workflows/cli_test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test CLI
```

Comment:
we can not use CLI here, we need a better name

Reply:
Emmm any ideas? What this action does?

Reply:
It's not just for these cases. I think we could have other test cases on this CI. So I think it's OK for this.

Reply:
ok, got it.